docs/docs/creating-a-source-plugin.md

```diff
@@ -413,7 +413,9 @@ query {
     id
     remoteImage {
       childImageSharp {
-        # fluid or fixed fields for optimized images
+        gatsbyImageData(
+          # fluid or fixed fields for optimized images
```

Comment:
Should this be

```
# arguments for optimized images
```

now?

(Sorry, not trying to be nit picky, learning about `gatsby-plugin-image` still :-/)

Reply:
That would work :D